backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/types/AcceptingStore.kt

```diff
@@ -7,6 +7,7 @@ data class AcceptingStore(
     val postalCode: String?,
     val street: String?,
     val houseNumber: String?,
+    val additionalAddressInformation: String?,
```

Comment:
Maybe add a todo here that this is required to be added in the database.

Reply:
There is #479 already which says that it should be added to the db